src/sql/base/browser/ui/table/formatters.ts

```diff
@@ -170,6 +143,55 @@ export function slickGridDataItemColumnValueWithNoData(value: any, columnDef: an
 	};
 }
 
+export function treeGridExpandableColumnFormatter<T>(formattingFunction: Slick.Formatter<T>) {
+	return (row: number | undefined, cell: any | undefined, value: any, columnDef: any | undefined, dataContext: any | undefined): string => {
+		const spacer = `<span style='display:inline-block;height:1px;width:${(15 * (dataContext['level'] - 1))}px'></span>`;
+
+		const innerCellContent = formattingFunction(row, cell, value, columnDef, dataContext);
+
+		if (dataContext['isParent']) {
+			if (dataContext.expanded) {
+				return `<div>${spacer}<span class='codicon codicon-chevron-down toggle' style='font-weight:bold;'></span>&nbsp; ${innerCellContent}</div>`;
+			} else {
+				return `<div>${spacer}<span class='codicon codicon-chevron-right toggle' style='font-weight:bold;'></span>&nbsp; ${innerCellContent}</div>`;
+			}
+		} else {
+			return `${spacer}${innerCellContent}`;
+		}
+	};
+}
+
+
+function createTextCell(value: any): string {
```

Comment:
Now that we're not using this in the new formatter you shouldn't need to separate this out